indicluster/app.py

```diff
@@ -211,15 +247,17 @@ def post(self):
         for metadata, indico_metadata in zip(objs, formatted_results):
             # save to database
             link, title, text = metadata
-            entry = Entry(
-                text=text,
-                title=title,
-                link=link,
-                indico=indico_metadata,
-                group=group,
-                rss_feed=url
-            )
-            session.add(entry)
+            already_exists = session.query(Entry).filter(Entry.link==link).first()
```

Comment:
Its worth noting if we add a field it won't update old links because of this. I think it still makes sense though.